### PR TITLE
Update PropelParamConverter.php

### PR DESCRIPTION
**Pull request by arffak**
add "query_method" option equivalence of "repository_method" in doctrine

### CODE REVIEW

**Comment by COil** [comment]
:+1:

File: Request/ParamConverter/PropelParamConverter.php [hunk 1]
```diff
@@ -50,6 +50,12 @@ class PropelParamConverter implements ParamConverterInterface
     protected $withs;
 
     /**
+     * name of method use to call a query method
+     * @var string
```
Comment (havvg) [inline Request/ParamConverter/PropelParamConverter.php]: There is blank line missing before the `@var`.

File: Request/ParamConverter/PropelParamConverter.php [hunk 2]
```diff
@@ -125,15 +131,24 @@ public function apply(Request $request, ParamConverter $configuration)
...
-            // find by criteria
-            if (false === $object = $this->findOneBy($classQuery, $request)) {
-                if ($configuration->isOptional()) {
-                    //we find nothing but the object is optional
-                    $object = null;
-                } else {
-                    throw new \LogicException('Unable to guess how to get a Propel object from the request information.');
+        $this->queryMethod = $queryMethod = isset($options['query_method']) ? $options['query_method'] : null;
+
+        if ($this->queryMethod != null && method_exists($classQuery, $this->queryMethod)) {
```
Comment (havvg) [inline Request/ParamConverter/PropelParamConverter.php]: CS: `if (null !== $this->queryMethod && method_exists(...)) {`

**Comment by havvg** [comment]
After applying the outstanding changes, could you please rebase and squash the commits?

**Comment by marcj** [comment]
Thanks, I merged and fix this CS myself :)

**Pull request merged**